src/marketplace/Marketplace.js

```diff
@@ -3,11 +3,61 @@
 import * as React from 'react';
 import { Link } from 'react-router-dom';
 import 'bootstrap/dist/css/bootstrap.min.css';
+import '../Main.css';
+import ChassisView from './ChassisView.js';
+import CannonsViews from './CannonsView.js';
+import CasusBlocksView from './CasusBlocksView.js';
+import JammersView from './JammersView.js';
+import TreadsView from './TreadsView.js';
+import AccessoriesView from './AccessoriesView.js';
+import ScannersView from './ScannersView.js';
+import MarketPlaceViewDataTypes from './MarketPlaceViewDataTypes.js';
```

Comment:
The import for the MarketPlaceViewDataTypes is incorrect. Here is what it should be:
`import type { MarketPlaceViewDataTypes } from './MarketPlaceViewDataTypes.js';`